src/Neo/SmartContract/Native/ContractManagement.cs

```diff
@@ -145,6 +145,19 @@ public ContractState GetContract(IReadOnlyStore snapshot, UInt160 hash)
             return snapshot.TryGet(key, out var item) ? item.GetInteroperable<ContractState>(false) : null;
         }
 
+        /// <summary>
+        /// Check if exists the deployed contract with the specified hash.
+        /// </summary>
+        /// <param name="snapshot">The snapshot used to read data.</param>
+        /// <param name="hash">The hash of the deployed contract.</param>
+        /// <returns>True if deployed contract exists.</returns>
+        [ContractMethod(Hardfork.HF_Echidna, CpuFee = 1 << 14, RequiredCallFlags = CallFlags.ReadStates)]
```

Comment:
BTW, I'm not sure we use values less than `1<<15` anywhere for methods. Even `GetMinimumDeploymentFee()` is `1<<15` and it has about the same overhead as this one.

Reply:
Should be cheaper because it doesn't process the contract and the manifest

Reply:
How about making it `1<<15` for now and moving this discussion to a separate issue? Because in general I agree with Roman, `GetMinimumDeploymentFee` logic is the same as the logic of `IsContract` in terms of storage access and price, but both these methods require less resources than some `GetContractByID`.

So to me it looks like we need to review/adjust all prices of native methods because right now some of them are imbalanced (the problem is somehow similar to #3510).

Reply:
But if the cost is the same, why a contract will use this call?

Reply:
Valid question. I've checked other contracts, looks like we do have values lower than `1<<15`, but they're rare, mostly in StdLib. We need to review these fees, there is definitely some imbalance here. For now, lower fee is OK to differentiate from `GetContract`. And we'll leave the question of why Policy invocations are more costly and why serialization/deserialization in StdLib is cheaper.

Reply:
Ref. https://github.com/neo-project/neo/issues/3894.